src/main/java/com/internet/shop/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.internet.shop.dao.impl;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.db.Storage;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Order;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order create(Order order) {
+        Storage.addOrder(order);
+        return order;
+    }
+
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        return Storage.orders.stream()
+                .filter(order -> order.getUserId().equals(userId))
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public Optional<Order> get(Long orderId) {
+        return Storage.orders.stream()
+                .filter(order -> order.getId().equals(orderId))
+                .findFirst();
+    }
+
+    @Override
+    public List<Order> getAll() {
+        return Storage.orders;
+    }
+
+    @Override
+    public boolean delete(Long orderId) {
+        return Storage.orders
+                .removeIf(order -> order.getId().equals(orderId));
```

Comment:
```suggestion
        return Storage.orders.removeIf(order -> order.getId().equals(orderId));
```
there is no need to split line into two separate, if it's not a stream